native/src/routes/JpalTracking.tsx

```diff
@@ -81,7 +81,7 @@ const JpalTracking = ({ navigation }: JpalTrackingProps): ReactElement => {
 
         <DescriptionContainer role='button' onPress={toggleTrackingEnabled}>
           <ThemedText>{t('allowTracking')}</ThemedText>
-          <SettingsSwitch value={!!trackingEnabled} onPress={toggleTrackingEnabled} />
+          <Switch value={!!trackingEnabled} onChange={toggleTrackingEnabled} />
```

Comment:
I think it should be `onValueChange`:
https://callstack.github.io/react-native-paper/docs/components/Switch/#onvaluechange